src/System.ServiceModel.Duplex/src/netcore50/project.json

```diff
@@ -2,7 +2,7 @@
   "frameworks": {
     "netcore50": {
       "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.0.1-rc3-24206-00",
+        "Microsoft.NETCore.Platforms": "1.0.1",
         "Microsoft.TargetingPack.Private.CoreCLR": "1.0.0-rc3-24206-00"
```

Comment:
Sounds good... as long as this will not bring in any pre release package dependency. Thanks!